client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/questions/index.tsx

```diff
@@ -18,9 +18,9 @@ export const Questions: FC = () => {
 				iconSize={ 12 }
 				target="_blank"
 				onClick={ () => {
-					recordTracksEvent(
-						'calypso_onboarding_site_migration_instructions_questions_happiness_engineer'
-					);
+					recordTracksEvent( 'calypso_site_migration_instructions_link_click', {
```

Comment:
Small nit, but it might be nice to have this in a function we can call in multiple places similar to https://github.com/Automattic/wp-calypso/blob/195b194e43a693303929e83a12b881f7be7c8bee/client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions-i2/index.tsx#L114.

Reply:
Thank your for the suggestion! I extracted to a separate file and implemented it here: https://github.com/Automattic/wp-calypso/pull/92845/commits/db152663122b4b65c283bcf89fd8576014d65d4e

Before I avoided it because of prop drilling, but I hadn't thought about extracting to another file. =)